lib/modules/platform/gitea/index.ts

```diff
@@ -173,6 +180,47 @@ async function lookupLabelByName(name: string): Promise<number | null> {
   return labelList.find((l) => l.name === name)?.id;
 }
 
+function getRepoUrl(
+  repo: helper.Repo,
+  gitUrl: GitUrlOption | undefined
+): string {
+  if (gitUrl === 'ssh') {
+    if (!repo.ssh_url) {
+      throw new Error(CONFIG_GIT_URL_UNAVAILABLE);
+    }
+    logger.debug({ url: repo.ssh_url }, `using ssh URL`);
```

Comment:
```suggestion
    logger.debug({ url: repo.ssh_url }, `using SSH URL`);
```

SSH is an abbreviation, so should be capitalized.